newIDE/app/src/EventsSheet/Toolbar.js

```diff
@@ -47,6 +48,11 @@ export class Toolbar extends PureComponent<Props> {
             src="res/ribbon_default/commentaireadd32.png"
             tooltip={t`Add a comment`}
           />
+          <ToolbarIcon
+            onClick={this.props.onToggleDisabledEvent}
+            src="res/ribbon_default/subeventadd32.png"
+            tooltip={t`Toggle disabled selected event`}
+          />
```

Comment:
These lines add a new button in the event sheet toolbar, I'm not sure we want to have yet another button in this toolbar, with the same icon as for "Add a sub event"